runtime/plugin/lsp.vim

```diff
@@ -0,0 +1,28 @@
+" TODO(tjdevries): Move this to autoload?
+
+execute('luafile ' . expand('<sfile>:h') . '/../lua/lsp/plugin.lua')
+
+function! LSP_Start() abort
+lua << EOF
+client_start()
+EOF
+endfunction
+
+" TODO: Automatically open file if not opened already
+function! LSP_References() abort
+lua << EOF
```

Comment:
Just like on #6789, I would say that it is vastly preferable if we put some of these finer points in the help docs, where people are likely to find them.